Tasks/GitHubPRComment/githubComment/task.json

```diff
@@ -60,6 +60,14 @@
             "defaultValue": "",
             "required": true,
             "helpMarkDown": "Get all the .json files including the ones in sub-folders"
+        },
+        {
+            "name": "keepCommentHistory",
+            "type": "boolean",
+            "label": "Keep comment history",
+            "defaultValue": "",
```

Comment:
Shouldn't be this an optional parameter with the default value `false`?

Reply:
The getSubFolders also is a boolean, and has the defaultValue as "", here is an example https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AzureCLIV1/task.json#L98-L105

Reply:
Ok, I'll set both to false.

Reply:
Done. Thanks!